test/integration/integration_test.go

```diff
@@ -66,9 +66,9 @@ func TestMain(m *testing.M) {
 	}
 	runner := util.MinishiftRunner{CommandPath: minishiftBinary}
 	if runner.IsCDK() {
-		godogTags += "~minishift-only"
+		godogTags += "~minishift-only ~b2d-only"
 	} else {
-		godogTags += "~cdk-only"
+		godogTags += "~cdk-only ~rhel-only"
```

Comment:
What about making it centos-only as for upstream users it will be easy to test with CentOS ISO and RHEL ISO is not available for upstream.